setup.py

```diff
@@ -75,6 +75,7 @@
         # have to pin it
         'django-filter~=1.0.0',
     ],
+    python_requires='~=3.4',
```

Comment:
You should also allow Python 3.5

Reply:
According to the [official Python Packaging User Guide](https://packaging.python.org/tutorials/distributing-packages/#python-requires), this would allow Python 3.4, 3.5, 3.6, ... anything below 4.

As we've discussed online, I'll change this to `python_requires='>=3.4, <3.6'` since we only officially support Python 3.4 and 3.5 right now.